### ISSUE TITLE
Don't uppercase "Read More" and other footer text in cards

### ISSUE THREAD

**Issue by tofumatt** [mozilla/addons-frontend#3190]
Go to https://addons-dev.allizom.org/en-US/android/addon/lastpass-password-manager/ and check out the all uppercase text in the footer cards:

<img width="451" alt="screenshot 2017-09-20 13 31 05" src="https://user-images.githubusercontent.com/90871/30643741-0d045e6a-9e08-11e7-9d1a-1887f083029a.png">
<img width="810" alt="screenshot 2017-09-20 13 31 03" src="https://user-images.githubusercontent.com/90871/30643740-0d031884-9e08-11e7-967a-31dd4f680a52.png">

There are plenty of issues with uppercasing text, so this should be regular case. We could *maybe* use small-caps but even they have accessibility problems so I'd rather not.

**Comment by tofumatt** [comment]
This should be a matter of just removing `font-variant-alternates: all-small-caps;` from the styles.

**Comment by Anshika-G** [comment]
@tofumatt  can I take this?

**Comment by tofumatt** [comment]
Hi @Anshika-G, yes this would be a good first bug. I'll mark it as assigned 😄

**Comment by Anshika-G** [comment]
thanks,
Actually, I'm a beginner, can u pls guide!

**Comment by tofumatt** [comment]
The CSS is in the `ui/components/Card/Card.scss` file under:

```scss
.Card-footer-link,
.Card-footer-text
```

You'll want to remove the `font-variant` property and attach before/after screenshots to your pull request 😄

**Comment by Anshika-G** [comment]
okay, Thanks!

**Comment by AlexandraMoga** [comment]
This was verified on AMO-dev FF56 (Win10 and Android 7.0)

Postfix screenshot:

![upper to low case](https://user-images.githubusercontent.com/31961530/31120111-fb25bef8-a83b-11e7-8bdb-98ae129411eb.png)

Note: should this change also affect the 'Log in to rate ...' banner? This is still displayed in upper-case.

**Comment by tofumatt** [comment]
Oh, good catch! Yes, yes we should. Could you file a new issue about that? Thanks.

- tofumatt
